build.gradle

```diff
@@ -0,0 +1,171 @@
+                          '-DPPX_BUILD_TESTS=FALSE',
+                          '-DBUILD_TESTS=FALSE' // Required for OpenXR
+                cppFlags '-std=c++20'
+                if (project.hasProperty('DXC_PATH')) {
+                  arguments "-DDXC_PATH=$DXC_PATH"
+                }
+            }
+        }
+        multiDexEnabled true
+    }
+
+    buildTypes {
+        release {
+            minifyEnabled false
+        }
+        debug {
+          applicationIdSuffix '.debug'
+          debuggable true
+        }
+    }
+
+    def defineFlavor = { sample_name ->
+      return {
+        dimension 'project'
+        it.buildConfigField "String", "sample_library_name", '"vk_' + sample_name + '"'
+        it.resValue         "string", "sample_library_name",  'vk_' + sample_name
+        it.resValue         "string", "app_label", sample_name
+      }
+    }
+
+    flavorDimensions 'device', 'project'
+    productFlavors {
+        triangle      defineFlavor('01_triangle')
+        cube          defineFlavor('04_cube')
+        cubexr        defineFlavor('04_cube_xr')
+        gbuffer       defineFlavor('16_gbuffer')
+        fishtornado   defineFlavor('fishtornado')
+        fishtornadoxr defineFlavor('fishtornado_xr')
+
+        mobile {
+            dimension 'device'
+            externalNativeBuild {
+                cmake {
+                    arguments '-DPPX_BUILD_XR=FALSE',
+                              '-DPPX_XR_QUEST=FALSE',
+                              '-DBUILD_LOADER=FALSE'
+                }
+            }
+            sourceSets {
+                main {
+                    manifest.srcFile 'src/android/AndroidManifestXR.xml'
+                }
+            }
+        }
+        openxr {
+            dimension 'device'
+            applicationIdSuffix '.xr'
+            versionNameSuffix '-xr'
+            externalNativeBuild {
+                cmake {
+                    arguments '-DPPX_BUILD_XR=TRUE',
+                              '-DPPX_XR_QUEST=FALSE',
+                              '-DBUILD_LOADER=TRUE',
+                              '-DBUILD_ALL_EXTENSIONS=TRUE'
+                    cppFlags '-DXR_USE_PLATFORM_ANDROID'
+                }
+            }
+            sourceSets {
+                main {
+                    manifest.srcFile 'src/android/AndroidManifestXR.xml'
+                }
+            }
+        }
+        quest {
+            dimension 'device'
+            applicationIdSuffix '.quest'
+            versionNameSuffix '-quest'
+            externalNativeBuild {
+                ndk {
+                    abiFilters 'arm64-v8a', 'armeabi-v7a'
+                }
+                cmake {
+                  arguments '-DPPX_BUILD_XR=TRUE',
+                            '-DPPX_XR_QUEST=TRUE',
+                            '-DBUILD_LOADER=FALSE',
+                            '-DBUILD_ALL_EXTENSIONS=FALSE',
+                            "-DOCULUS_OPENXR_SDK=${oculusOpenxrSdkLocation}"
```

Comment:
The Quest OpenXR loader is currently omitted from this PR. Can you link the imported loader?

My change for this is [here](https://github.com/google/bigwheels/pull/90/files#diff-3bd332c63c934c2fa3d25499843e6da0cbc9fd176e2ac9a798a5a6d4c650ae20).